File: ProductOrders/Scripts/models/orderIdentification.js
var OrderIdentification = function (orderId, customerId, customerName, orderStatus) {
    var self = this;

    self.orderId = ko.observable(orderId);
    self.customerId = ko.observable(customerId);
    self.customerName = ko.observable(customerName);
    self.orderStatus = ko.observable(orderStatus);

    return self;
};

File: ProductOrders/Models/OrderIdentification.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProductOrders.Models
{
    public class OrderIdentification
    {
        public string OrderId { get; set; }
        public string CustomerId { get; set; }
        public string CustomerName { get; set; }
        public string OrderStatus { get; set; }        
    }
}